### PR TITLE
Fix invalid response type in CallBundle

### PR DESCRIPTION
**Pull request by ngotchac**
This PR fixes the response type of the `revert` value for the `CallBundle` RPC call. It was encoded with `string(revert)`, `revert` being a `[]byte` array, which produces invalid UTF-8 symbols. It makes it impossible to decipher the actual `revert` reason.
Using `common.Bytes2Hex` fixes the issue.

It also fixes an issue in which the actual `ctx` objects, created with a timeout in `CallBundle` and `EstimateGasBundle` were never used.

### CODE REVIEW

File: internal/ethapi/api.go [hunk 1]
```diff
@@ -2377,12 +2382,10 @@ func (s *BundleAPI) CallBundle(ctx context.Context, args CallBundleArgs) (map[st
 			jsonResult["error"] = result.Err.Error()
 			revert := result.Revert()
 			if len(revert) > 0 {
-				jsonResult["revert"] = string(revert)
+				jsonResult["revert"] = "0x" + common.Bytes2Hex(revert)
```
Comment (Ruteri) [inline internal/ethapi/api.go:2385]: Wouldn't hexutil.Encode fit better here?

**Comment by Ruteri** [comment]
Since this is changes the RPC we will need to wait for the next release.
Thanks for the PR!

File: internal/ethapi/api.go (hunk 1 above)
Comment (ngotchac) [reply internal/ethapi/api.go]: Yep you're right, I replaced `common.Bytes2Hex` by `hexutil.Encode` where appropriate

**Comment by ngotchac** [comment]
BTW, you might want to add a `make lint` job to the Github Workflows, which would have caught the `context` missuses :)

**Pull request closed without merging**